.dagger/sdk_typescript.go

```diff
@@ -21,10 +21,10 @@ const (
 	typescriptRuntimeSubdir    = "sdk/typescript/runtime"
 	typescriptGeneratedAPIPath = "sdk/typescript/api/client.gen.ts"
 
-	nodeVersionMaintenance = "18"
-	nodeVersionLTS         = "20"
+	nodeVersionMaintenance = "20.18.1"
+	nodeVersionLTS         = "22.11.0"
```

Comment:
Having discussed this with @TomChv sync, my understanding is that the reason why we have both versions here is so that we test with the previous LTS as well as the current LTS.

If we are still finding this useful, then we should rename these two variables so that they are self-explanatory:
- `nodePreviousLTS`
- `nodeCurrentLTS`

When v24 comes out in 2024, that will become the current LTS version, and v22 will become the previous one.

Reply:
Done in [696ceef](https://github.com/dagger/dagger/pull/8823/commits/696ceefc8dd7193550bec96a4996761df361f34a)